src/dstore/dstore_base.c

```diff
@@ -291,6 +353,8 @@ static int dstore_io_op_init_and_submit(struct dstore_obj *obj,
 	dassert(op_type == DSTORE_IO_OP_WRITE ||
 		op_type == DSTORE_IO_OP_READ);
 
+	perfc_trace_inii(PFT_DSTORE_IO_OP_INIT_AND_SUBMIT, PEM_DSTORE_TO_NFS);
+
```

Comment:
Suggestion: do we need to trace op_type?

Reply:
As of now, only read and write ops are allowed inside dstore_io_op_init_and_submit, and we have traces in both of those functions, so the call flow will be clear. Hence not needed.

Reply:
ok